ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -154,29 +189,35 @@ async def test_analyze(ofrak_server, aiohttp_client, hello_world_elf):
     create_resp = await client.post(
         "/create_root_resource", params={"name": "test"}, data=hello_world_elf
     )
-    body = await create_resp.json()
-    resp = await client.post(f"/{body['id']}/analyze")
+    create_body = await create_resp.json()
+    resp = await client.post(f"/{create_body['id']}/analyze")
     assert resp.status == 200
+    resp_body = await resp.json()
+    assert resp_body is not None
 
 
 async def test_identify(ofrak_server, aiohttp_client, hello_world_elf):
     client = await aiohttp_client(ofrak_server._app)
     create_resp = await client.post(
         "/create_root_resource", params={"name": "test"}, data=hello_world_elf
     )
-    body = await create_resp.json()
-    resp = await client.post(f"/{body['id']}/identify")
+    create_body = await create_resp.json()
+    resp = await client.post(f"/{create_body['id']}/identify")
     assert resp.status == 200
+    resp_body = await resp.json()
+    assert resp_body["modified"][0]["id"] == create_body["id"]
 
 
 async def test_data_summary(ofrak_server, aiohttp_client, hello_world_elf):
     client = await aiohttp_client(ofrak_server._app)
     create_resp = await client.post(
         "/create_root_resource", params={"name": "test"}, data=hello_world_elf
     )
-    body = await create_resp.json()
-    resp = await client.post(f"/{body['id']}/data_summary")
+    create_body = await create_resp.json()
+    resp = await client.post(f"/{create_body['id']}/data_summary")
     assert resp.status == 200
+    resp_body = await resp.json()
+    assert resp_body["modified"][0]["id"] == create_body["id"]
```

Comment:
Since we can import OFRAK from the test, it might make sense to do the same computations with OFRAK (_i.e._, run the `DataSummaryAnalyzer`) and compare the results to make sure the server is sending back the same responses we'd calculate locally.